src/LiveComponent/doc/index.rst

```diff
@@ -2531,10 +2528,11 @@ that will change if the component should be totally re-rendered:
     <div {{ attributes }}>
         <!-- ... -->
 
-        {{ component('TodoFooter', {
-            count: todos|length,
-            'data-live-id': 'todo-footer-'~todos|length
-        }) }}
+        <twig:TodoFooter 
+            :count="todos|length"
+            :data-live-id="'todo-footer-'~todos|length"
```

Comment:
Same question as below: does this work:

```suggestion
            data-live-id="todo-footer-{{ todos|length }}"
```

Reply:
Another option. I wish there were a shorthand to the format filter, I use this syntax for clarity instead of ~, although it often ends up being longer.

```twig
:data-live-id="'todo-footer-%d'|format(todos|length)"
```